code/Collector/PhpInfoCollector.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace LeKoala\DebugBar\Collector;
+
+/**
+ * Extends the default PhpInfoCollector to trim long PHP version numbers
+ */
+class PhpInfoCollector extends \DebugBar\DataCollector\PhpInfoCollector
+{
+    public function collect()
+    {
+        $metrics = parent::collect();
+        $metrics['version'] = $this->trimVersion($metrics['version']);
+        return $metrics;
+    }
+
+    /**
+     * Given a PHP_VERSION constant value, trim any "extra" meta information from the end, returning the major, minor
+     * and patch release of the version. Will fall back to returning the input if the matching fails.
+     *
+     * @param string $version
+     * @return string
+     */
+    protected function trimVersion($version)
+    {
+        preg_match('/^(\d+\.\d+\.\d+)/', $version, $matches);
```

Comment:
Pretty sure you don't need the capturing group here - but it's so minor I won't block merge 🙂

Reply:
TIL

Reply:
Turns out I don't have the authority to merge (or therefore block it I guess).